0000-report-inboxes.md

```diff
@@ -0,0 +1,82 @@
+   * This is akin to the current 0.19.3 admin report view, and would allow admins to still keep an eye on mod actions on their instance if they wish
+    
+The UI wouldn't need to change for mods, but for admins, there would be a new selection at the top of the reports page (the "mod reports" tab would only be visible if the admin is also a mod in any community):
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/cc4ad68c-6e85-4cd9-b324-131c06951cb3)
+
+### Resolving reports should be more granular
+
+* Reports in the "admin reports" tab can only be manually resolved for admins of the local instance
+   * To reduce overhead, **banning the reported user on the user's home instance + removing reported content should automatically resolve reports for remote admins as well**.
+* Reports in the "mod reports" tab should be manually resolved by relevant mods (including admins, if they are explicit mods in the relevant community).
+   * To reduce overhead, **admins banning the reported user on the community instance OR the user's home instance + removing reported content should automatically resolve reports for mods as well**
+* Admins could still resolve reports in the "all reports" tab
+   * If it's not an admin report, and not a mod report from a community the admin explicitly moderates, then there should be an additional warning/confirmation when resolving a report here. This is to prevent cases of admins accidentally preventing mods from moderating according to their own community rules.
+
+To further clarify automatic resolution of reports: in any case where there is no further action possible, the report should be automatically resolved.
+  
+### Mods should be able to escalate reports to admins
+
+This would generate a corresponding report in the admin inbox.
+
+# Reference-level explanation
+
+* In the UI, changes are needed for both reporting as well as the reports inbox views
+* In the database and API, we should split reports by intended audience
+* Federation needs to be changed as well in order to allow distinguishing the report target audience
+
+# Drawbacks
+
+It might make reporting slightly more confusing for end users - the mod/admin distinction might not be fully clear to all.
+
+# Rationale and alternatives
+
+Alternatively, we could make reporting **even more** granular. It would be possible to allow users to select only a specific instances admins as the intended report audience, for example.
+However, I think this has several downsides:
+* Makes the report UI even more confusing
+* Potentially takes away valuable information from other admins (imagine a user only reports CSAM to their own instances admins, while leaving the offending post authors home admins in the dark)
+
+# Prior art
+
+Most other social networks allow users to select whether they are reporting a violation of community rules, or site rules as whole.
+
+# Unresolved questions
+
+Does ActivityPub properly support splitting up reports like this?
```

Comment:
We can set the `to` and `audience` fields either to the instance url or the community url depending on the target.

https://github.com/LemmyNet/lemmy/blob/main/crates/apub/assets/lemmy/activities/community/report_page.json#L3